src/sentry/api/endpoints/organization_events.py

```diff
@@ -197,15 +207,19 @@ def get(self, request, organization):
 
         lookup_keys = []
         non_tag_lookup_keys = []
-        for key in request.GET.getlist('keys'):
+
+        # TODO(lb): some validation should probably occur but I'm not clear what
+        keys = request.GET.getlist('keys')
```

Comment:
We could make sure that the tag keys match the pattern used by /store endpoint which is `^[a-zA-Z0-9_\.:-]+$`